NAMESPACE

```diff
@@ -11,8 +11,8 @@ S3method(ctype,SHEETS_CELL)
 S3method(ctype,character)
 S3method(ctype,default)
 S3method(ctype,list)
-S3method(format,sheets_meta)
-S3method(print,sheets_meta)
+S3method(format,sheets_Spreadsheet)
+S3method(print,sheets_Spreadsheet)
```

Comment:
This foreshadows an experiment here, which is to start creating companion S3 classes for some of the more import **schemas** for the Sheets API. `Spreadsheet` is an example of that. The existing `sheets_meta` class happened to be exactly this for the `Spreadsheet` schema.